client/src/app/components/Footer.jsx

```diff
@@ -8,15 +9,16 @@ const Footer = () => (
     <div className="wrapper footer__wrapper">
       <span className="footer__text">
         <Link className="footer__text footer__link" to="/about">
-          About
+          <FormattedMessage id="footer.about" />
         </Link>
         <span className="footer__text-divider"> | </span>
         <Link className="footer__text footer__link" to="/privacy-policy">
-          Privacy & Terms
+          <FormattedMessage id="footer.privacy" />
         </Link>
       </span>
       <span className="footer__text footer__copyrights">
-        &copy; Copyrights 2019
+        &copy;
```

Comment:
you can move the copyright sign into the translation text

Reply:
I can't, cause react intl renders this as a string

"$copy Copyrights 2019"